src/test/java/edu/hm/hafner/edu/hm/hafner/analysis/IssueTest.java

```diff
@@ -0,0 +1,271 @@
+package edu.hm.hafner.edu.hm.hafner.analysis;
+
+import org.junit.jupiter.api.Test;
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.Priority;
+import edu.hm.hafner.edu.hm.hafner.analysis.edu.hm.hafner.analysis.assertions.IssueSoftAssertions;
+
+@SuppressWarnings({"NonBooleanMethodNameMayNotStartWithQuestion", "JUnitTestMethodWithNoAssertions"})
```

Comment:
JUnitTestMethodWithNoAssertions: In IntelliJ lassen sich die AssertJ Assertions registrieren

Reply:
NonBooleanMethodNameMayNotStartWithQuestion: sic, should entfernen